src/localize/languages/es.json

```diff
@@ -0,0 +1,76 @@
+{
+  "scheduler": "Programación",
+  "actions": {
+    "add": "añadir dispositivo",
+    "cancel": "cancelar",
+    "next": "siguiente",
+    "save": "guardar",
+    "delete": "borrar"
+  },
+  "instructions": {
+    "no_group_selected": "Selecciona un grupo",
+    "no_entity_selected": "Selecciona una entidad",
```

Comment:
Desde el contexto de la version en ingles: 
"Selecciona un grupo primero"
"Selecciona una entidad primero"
Es una advertencia si se trata de crear un item nuevo sin elegir grupo o entidad.

___________________________________________________________________________

From the context of the English version:
"Selecciona un grupo primero"
"Selecciona una entidad primero"
It is a warning if you try to create a new item without choosing a group or entity.